python/ray/serve/_private/replica.py

```diff
@@ -165,6 +165,21 @@ def __init__(
             RAY_SERVE_GAUGE_METRIC_SET_PERIOD_S,
         )
 
+        self.set_autoscaling_config(autoscaling_config)
+
+    def start(self):
+        """Start periodic background tasks."""
+        self._metrics_pusher.start()
+
+    def shutdown(self):
+        """Stop periodic background tasks."""
+        self._metrics_pusher.shutdown()
+
+    def set_autoscaling_config(self, autoscaling_config: AutoscalingConfig):
```

Comment:
Can the `autoscaling_config` argument be `None`? The `if self._autoscaling_config` conditional below implies that's the case.

Could you change the `autoscaling_config` type hint to `Optional[AutoscalingConfig]` if so?